package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@charmverse/core",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "type": "module",
+  "scripts": {
+    "build": "npm run prisma:generate && tsc",
+    "prisma:generate": "npx prisma generate --schema=./src/prisma/schema.prisma",
+    "postinstall": "npm run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/charmverse/common.git"
+  },
+  "keywords": [
+    "charmverse"
+  ],
+  "author": "",
+  "license": "ISC",
```

Comment:
remove this and add the LICENSE file from charmverse app. Not sure what the license will be for permissiosn api yet, but it's closed source so we can add later